.github/workflows/foodies-backend-ci.yml

```diff
@@ -0,0 +1,156 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Foodies BE CI
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
```

Comment:
Yeah I was thinking that too. To me, I didn't see a point to run CI for PRs that aren't merging into master since that's the branch we care about at the end of the day.